src/ResourceManager/ServiceBus/Commands.ServiceBus/Models/AuthorizationRuleAttributes.cs

```diff
@@ -1,78 +0,0 @@
-﻿// ----------------------------------------------------------------------------------
-//
```

Comment:
Why are all these files getting deleted?

Reply:
added prefix of PS to the models.

Reply:
I don't see a replacement of these two files anywhere with PS prefix.

Reply:
Approved offline.